requests/caps.js

```diff
@@ -101,9 +101,27 @@ export const getAbandonoMensal = async (municipioIdSus) => {
   }
 };
 
-export const getAbandonoCoortes = async (municipioIdSus) => {
+export const getPerfilAbandono = async (municipioIdSus) => {
+  try {
+    const endpoint = "/abandono/resumo?municipio_id_sus=" + municipioIdSus;
+
+    const { data } = await axiosInstance.get(endpoint);
+
+    return data;
+  } catch (error) {
+    console.log('error', error.response.data);
+  }
+};
+
+export const getAbandonoCoortes = async ({
+  municipioIdSus,
+  periodo,
+  estabelecimento
+}) => {
```

Comment:
Acho que vale a pena mudar também o nome dos parâmetros da função de requisição para o plural, de forma que quem use essa função, eventualmente, entenda que é possível passar para ela mais de um valor de período ou estabelecimento numa mesma requisição.

Caso mude os nomes dos parâmetros dessa função, não esquecer de atualizar os nomes na chamada dessa função na página de abandono.

```suggestion
export const getAbandonoCoortes = async ({
  municipioIdSus,
  periodos,
  estabelecimentos
}) => {
```